api/envoy/type/v3/path_transformations.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package envoy.type.v3;
```

Comment:
Final nit from me: maybe move this into an HTTP namespace? Perhaps envoy.type.http.v3 or something? Whatever you think is best.